pages/mission.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: otherpage
+title: Mission
+---
+
+The MDAnalysis community is dedicated to advancing computational molecular science by developing open-source tools for data processing and analysis. We foster an inclusive, collaborative environment where everyone is welcome.
+
+We follow our [Code of Conduct]({{site.baseurl}}/pages/conduct/) to ensure a welcoming space for all contributors. Our primary goal is to produce high-quality, trusted software for scientists in academia and industry.
+
+## What We Do
+
+- **Develop Open-Source Tools:** We create and maintain projects that support data analysis in computational molecular sciences.
+- **Empower Users & Developers:** We help researchers leverage our tools while encouraging them to become contributors.
+- **Uphold [FAIR principles][]:** We ensure our tools follow the principles of Findability, Accessibility, Interoperability, and Reusability.
+- **Promote Reproducibility & Best Practices:** We prioritize correctness, transparency, and reproducibility in scientific software development.
+
+Our central package, [MDAnalysis]({{ site.github.repo }}), provides robust tools for analyzing computer simulations of molecular systems.
+
+We believe that scientific software should be **open, reliable, and accessible** to all. Through documentation, training, and an engaged community, we equip researchers with the skills to use and contribute to our ecosystem.
```

Comment:
We voted on the Mission statement. Please keep as it was for now.

If you want to propose improvements then that's a different thing that we need to discuss separately.